src/lib/storageCylinder.ts

```diff
@@ -0,0 +1,58 @@
+import { Knex } from 'knex';
+import { knexController } from '../database/database';
+import {
+  CreateStorageCylinderBody,
+  StorageCylinder,
+} from '../types/storageCylinder.types';
+import { log } from './log';
+
+export const getStorageCylinder = async (
+  trx: Knex.Transaction,
+  id: number
+): Promise<StorageCylinder> => {
+  const storageCylinderQuery = await trx<StorageCylinder>('storage_cylinder')
+    .where('id', id)
+    .first(
+      'id',
+      'gas_id AS gasId',
+      'volume',
+      'name',
+      'max_pressure AS maxPressure'
+    );
+
+  if (storageCylinderQuery === undefined) {
+    log.error('Storage cylinder not found');
+    throw new Error('Storage cylinder not found');
+  }
+
+  const sc: StorageCylinder = JSON.parse(JSON.stringify(storageCylinderQuery));
+  return sc;
```

Comment:
```suggestion
  return JSON.parse(JSON.stringify(storageCylinderQuery));
```
Eikös ton pitäisi olla ok?